.github/workflows/ci.yml

```diff
@@ -141,7 +139,11 @@ jobs:
         id: 'prepare-test-environment'
         if: ${{ matrix.testEnv.shouldCreate }}
         env: ${{ matrix.testEnv.envVars }}
-        run: 'pnpm --filter="${{ matrix.projectName }}" ${{ matrix.testEnv.start }}'
+        run: |
+          if [[ '${{ matrix.testType }}' == 'unit:php' || '${{ matrix.testType }}' == 'api' ]]; then
```

Comment:
TBD: in this PR rework in favour of 'build_type' input for the `setup-woocommerce-monorepo` action.